### PR TITLE
Change path during publishing

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

File: eng/pipelines/ci.yml [hunk 1]
```diff
@@ -99,6 +99,7 @@ jobs:
         - pwsh: |
             $currentVersion = node -p -e "require('./src/package.json').version";
             $currentVersion="$currentVersion-$(Build.BuildNumber)";
+            cd src
```
Comment (catalinaperalta) [inline eng/pipelines/ci.yml:102]: Why are we doing this?

Comment (catalinaperalta) [reply eng/pipelines/ci.yml:102]: To only publish relevant generator files?

Comment (jhendrixMSFT) [reply eng/pipelines/ci.yml:102]: The package.json for this project is under the `src` directory.

Comment (catalinaperalta) [reply eng/pipelines/ci.yml:102]: Ah got it. Thank you!

**Pull request merged**